osidb/filters.py

```diff
@@ -276,6 +274,15 @@ class FlawFilter(DistinctFilterSet, IncludeFieldsFilterSet, ExcludeFieldsFilterS
     )
     cvss_scores__cvss_version = CharFilter(field_name="cvss_scores__version")
 
+    # Emptiness filters
+    cve_id__isempty = EmptyOrNullStringFilter(field_name="cve_id")
+    cve_description__isempty = EmptyOrNullStringFilter(field_name="cve_description")
+    statement__isempty = EmptyOrNullStringFilter(field_name="statement")
+    mitigation__isempty = EmptyOrNullStringFilter(field_name="mitigation")
+    owner__isempty = EmptyOrNullStringFilter(field_name="owner")
+    cwe_id__isempty = EmptyOrNullStringFilter(field_name="cwe_id")
+    cvss_scores__isempty = BooleanFilter(field_name="cvss_scores", lookup_expr="isnull")
```

Comment:
This only checks whether some CVSS is attached to a Flaw without any context about version or issuer, right?

I am not 100% sure about the usecase but I would imagine users would like to explicitly list only Flaws without RH CVSS v3 judging by the wording in the Jira issue. :thinking:

Reply:
Okay, but then I would rename the filter to something like `cvss3__isempty` to avoid confusion, as if I use the name of a field + `__isempty` I would expect emptiness on the whole field, not only an arbitrary subset. How does that sound to you?

Reply:
Yeah, definitely, we need to be explicit here. In the future, there will be also cvss4, etc. Also not sure, whether they are interested only in RH cvss3 or NVD cvss3 as well :thinking:

Reply:
I will go ahead and just add all of the possibilities